ci-operator/step-registry/interop-tests/ocs-tests/interop-tests-ocs-tests-commands.sh

```diff
@@ -18,6 +18,12 @@ LOGS_FOLDER="${ARTIFACT_DIR}/ocs-tests"
 LOGS_CONFIG="${LOGS_FOLDER}/ocs-tests-config.yaml"
 CLUSTER_PATH="${ARTIFACT_DIR}/ocs-tests"
 
+#
+# Remove the ACM Subscription to allow OCS interop tests full control of operators
```

Comment:
What if the cluster don't have ACM Subscription ?
the `get` command will fail, and the script will abort

Reply:
I made changes to address this.